server/app/main.py

```diff
@@ -19,3 +27,34 @@ async def mlmd_push(info: Request):
         'data':req_info
     }
 
+@app.get("/mlmd_pull",response_class=HTMLResponse)
+async def mlmd_pull(request: Request):
+    if os.path.exists("/cmf-server/data/mlmd"):
+        query = cmfquery.CmfQuery("/cmf-server/data/mlmd")
+        json_payload = query.dumptojson("Test-env")
+        return json_payload
+    else:
+        print("No mlmd file submitted")
+        json_payload=""
+        return json_payload
+
+
+@app.get("/display_executions",response_class=HTMLResponse)
+async def display_exec(request: Request):
+    if os.path.exists("/cmf-server/data/mlmd"):
+        execution_df=index("/cmf-server/data/mlmd")
+        query = cmfquery.CmfQuery("/cmf-server/data/mlmd")
+        json_payload = query.dumptojson("Test-env")
```

Comment:
There could be many pipelines . We have an api to get the pipeline names. use that to get all the pipeline names and query each pipeline individually at display. create separate tables for each pipeline